kit/preprocess.js

```diff
@@ -87,6 +91,30 @@ export const docstringPreprocess = {
 				svelteComponent += ` returnType={${JSON.stringify(code)}} `;
 			}
 
+			if (docstringBody.match(REGEX_YIELDESC)) {
+				const yieldDesc = docstringBody.match(REGEX_YIELDESC)[1];
+				const { code } = await mdsvexPreprocess.markup({ content: yieldDesc, filename });
+				svelteComponent += ` returnDescription={${JSON.stringify(code)}} `;
+			}
+
+			if (docstringBody.match(REGEX_YIELDTYPE)) {
+				const yieldType = docstringBody.match(REGEX_YIELDTYPE)[1];
+				const { code } = await mdsvexPreprocess.markup({ content: yieldType, filename });
+				svelteComponent += ` returnType={${JSON.stringify(code)}} isYield={true} `;
+			}
+
+			if (docstringBody.match(REGEX_RAISEDESC)) {
+				const raiseDesc = docstringBody.match(REGEX_RAISEDESC)[1];
+				const { code } = await mdsvexPreprocess.markup({ content: raiseDesc, filename });
+				svelteComponent += ` raiseDescription={${JSON.stringify(code)}} `;
+			}
+
+			if (docstringBody.match(REGEX_RAISETYPE)) {
+				const raiseType = docstringBody.match(REGEX_RAISETYPE)[1];
+				const { code } = await mdsvexPreprocess.markup({ content: raiseType, filename });
+				svelteComponent += ` raiseType={${JSON.stringify(code)}} `;
+			}
```

Comment:
Eventually, something like a key/value map for the properties and generating the component string at once seems cleaner (rather than multiple `svellteComponent += ...` followed by a closing tag)